### PR TITLE
[bitnami/redis] Fix readiness script typo

### PR DESCRIPTION
**Pull request by migruiz4**
Signed-off-by: Miguel Ruiz <miruiz@vmware.com>

Same fix as applied here: #9279

**Checklist**

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the values.yaml and added to the README.md using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [X] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])
- [X] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/redis/templates/health-configmap.yaml [hunk 1]
```diff
@@ -33,7 +33,7 @@ data:
 {{- end }}
         ping
     )
-    if [ $? == 124 ]; then
+    if [ "$?" -eq "124" ]; then
```
Comment (miguelaeh) [inline bitnami/redis/templates/health-configmap.yaml:36]:
```suggestion
    if [[ "$?" -eq "124" ]]; then
```

File: bitnami/redis/templates/health-configmap.yaml [hunk 2]
```diff
@@ -63,7 +63,7 @@ data:
...
```
Comment (miguelaeh) [inline bitnami/redis/templates/health-configmap.yaml:66]:
```suggestion
    if [[ "$?" -eq "124" ]]; then
```

File: bitnami/redis/templates/health-configmap.yaml [hunk 3]
```diff
@@ -97,7 +97,7 @@ data:
...
```
Comment (miguelaeh) [inline bitnami/redis/templates/health-configmap.yaml:100]:
```suggestion
    if [[ "$?" -eq "124" ]]; then
```

File: bitnami/redis/templates/health-configmap.yaml [hunk 4]
```diff
@@ -141,7 +141,7 @@ data:
...
```
Comment (miguelaeh) [inline bitnami/redis/templates/health-configmap.yaml:144]:
```suggestion
    if [[ "$?" -eq "124" ]]; then
```

File: bitnami/redis/templates/health-configmap.yaml [hunk 5]
```diff
@@ -169,7 +169,7 @@ data:
...
```
Comment (miguelaeh) [inline bitnami/redis/templates/health-configmap.yaml:172]:
```suggestion
    if [[ "$?" -eq "124" ]]; then
```

**Review by miguelaeh — CHANGES_REQUESTED** [review]

**Comment by migruiz4** [comment]
@miguelaeh Could you please provide more context on your change request?

I think the standard POSIX `[` is fine to check a number matches another.

**Review by miguelaeh — APPROVED** [review]
LGTM

**Pull request merged**